lib/internal/process/promises.js

```diff
@@ -43,7 +48,7 @@ function setupPromises(scheduleMicrotasks) {
         hasBeenNotifiedProperty.set(promise, true);
         if (!process.emit('unhandledRejection', reason, promise)) {
           // Nobody is listening.
-          // TODO(petkaantonov) Take some default action, see #830
```

Comment:
Naming nit: would it be possible to switch this from `onPromiseGC` to `trackPromise`, for symmetry?